packages/apostrophe-widgets/accordion/src/accordion.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { createRoot } from 'react-dom/client';
+import "@utrecht/component-library-css";
+import "@utrecht/design-tokens/dist/root.css";
+import { AccordionProvider } from "@utrecht/component-library-react";
+import RenderContent from '../../../ui/src/rte-formatting/rte-formatting.jsx'
```

Comment:
Is de extensie `.jsx` nodig in deze regel? Dat wilde in de automatische build nog wel eens fouten geven namelijk.